tests/PySys/tedge_mapper_c8y/tedge_mapper_c8y_child_device_measurement/run.py

```diff
@@ -0,0 +1,47 @@
+from environment_c8y import EnvironmentC8y
+
+import time
+import json
+import os
+
```

Comment:
Can you maybe also add a description of what the test is doing here? That way its easier to have an overview